tests/test_schemas.py

```diff
@@ -0,0 +1,172 @@
+import os
+import logging
+
+import pandas as pd
+import pytest
+
+from schematic.schemas import df_parser
+
+logging.basicConfig(level=logging.DEBUG)
+logger = logging.getLogger(__name__)
```

Comment:
**Random thought:** Is there precedent to configuring the logger in every Python file/submodule? Couldn't we define these two lines once in the root `__init__.py` file (one in `schematic/` and another in `tests/`) and import it in submodules? If so, this could happen in another PR.

Reply:
@BrunoGrandePhD: I think the reason for creating a logger in every (sub)module is so the logging module knows which (sub)module raised the logger warning/error. If the root logger (say in `schematic/__init__.py`) is always imported and used for logging then we'd see warnings like `schematic: logger message` rather than the specific (sub)module (example: `schematic/manifest/generator.py: logger message`).

This [SO answer](https://stackoverflow.com/questions/15727420/using-logging-in-multiple-modules) also recommends defining a logger within every (sub)module.

Reply:
But aren't these log messages intended for the users (and not us, developers)? Why do they have to know which submodule is emitting the log message?

Reply:
Wearing my user hat on, I'd agree it's easier to define the logger level in one place and have it be in effect across the codebase.

@BrunoGrandePhD are you concerned about having to maintain/change the logger code in multiple places across the codebase (w/ potential for logging levels to diverge)?

@sujaypatil96 do you think the source (sub)module is useful for developers?